docs/building-spark.md

```diff
@@ -12,7 +12,7 @@ redirect_from: "building-with-maven.html"
 ## Apache Maven
 
 The Maven-based build is the build of reference for Apache Spark.
-Building Spark using Maven requires Maven 3.3.9 or newer and Java 8+.
+Building Spark using Maven requires Maven 3.5.4 and Java 8.
```

Comment:
Maven 3.5.4 is the latest one and there is no newer version for now. We cannot guarantee the newer Maven work or not.